pkg/prometheus/promcfg.go

```diff
@@ -657,6 +661,10 @@ func (cg *ConfigGenerator) generatePodMonitorConfig(
 	if ep.Scheme != "" {
 		cfg = append(cfg, yaml.MapItem{Key: "scheme", Value: ep.Scheme})
 	}
+	// Defaults to true, so only add when value is set to false
+	if ep.FollowRedirects != nil && !ptrToBool(ep.FollowRedirects) {
+		cfg = append(cfg, yaml.MapItem{Key: "follow_redirects", Value: false})
+	}
```

Comment:
I would only check if the value isn't nil and if so, pass the value directly. What do you think?

```suggestion
	if ep.FollowRedirects != nil {
		cfg = append(cfg, yaml.MapItem{Key: "follow_redirects", Value: *ep.FollowRedirects})
	}
```

Reply:
i have some local changes to commit ( basically the tests ) but i can add that. 

I did not add it because the value `true` is the default but i guess better to be explicit

Reply:
done